recipes/libkml/all/conanfile.py

```diff
@@ -35,18 +37,13 @@ def source(self):
     def build(self):
         for patch in self.conan_data["patches"][self.version]:
             tools.patch(**patch)
-        cmake = self._configure_cmake()
-        cmake.build()
-
-    def _configure_cmake(self):
-        cmake = CMake(self)
-        cmake.configure(build_folder=self._build_subfolder)
-        return cmake
+        self._cmake = CMake(self)
```

Comment:
Using `_configure_cmake` was okay, I just meant caching cmake inside `_configure_cmake`
e.g. https://github.com/conan-io/conan-center-index/blob/935fe1ce7cf8f1092f12c70c45dbe0baa84c0798/recipes/bullet3/all/conanfile.py#L54:L56

Reply:
This should support separate build and package steps

Reply:
Oups, ok I see. Fixed.